integreat_cms/static/src/js/pois/poi-actions.ts

```diff
@@ -15,6 +15,9 @@ window.addEventListener("load", () => {
       // Reset to initial value
       longitude.value = longitude.dataset.initial;
       latitude.value = latitude.dataset.initial;
+
+      longitude.dispatchEvent(new Event("input"));
+      latitude.dispatchEvent(new Event("input"));
```

Comment:
```suggestion
      longitude.dispatchEvent(new Event("focusout"));
      latitude.dispatchEvent(new Event("focusout"));
```
I think we can just dispatch the `focusout` event here - then, we only need one event listener on the other side...